package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "hollowbot",
+  "version": "1.0.0",
+  "description": "Chatops for Hollowverse",
+  "main": "src/activate.ts",
+  "license": "Unlicense",
+  "scripts": {
+    "activate-dev": "nodemon --exec ts-node src/activate.ts",
+    "activate": "ts-node src/activate.ts",
+    "lint": "tslint '**/*.t{s,sx}' -e '**/node_modules/**' --fix",
```

Comment:
Could you update this to make it the same as what's on our main repo? I've built commonpkg so that it doesn't mess with a value if it already exists, so commonpkg will not update this for you.